librz/core/disasm.c

```diff
@@ -1815,9 +1815,10 @@ static void ds_show_functions(RDisasmState *ds) {
 	}
 	bool demangle = rz_config_get_b(core->config, "bin.demangle");
 	bool keep_lib = rz_config_get_b(core->config, "bin.demangle.libs");
-	bool showSig = ds->show_fcnsig && ds->show_calls;
-	bool call = rz_config_get_b(core->config, "asm.calls");
+	bool fcnsig = ds->show_fcnsig;
+	// bool call = rz_config_get_b(core->config, "asm.calls");
```

Comment:
Regarding `asm.calls`, I think usage of it in this part of the code was wrong, since it should affect only the comments in the disasm for instructions like `call sym.some_func` it should add the comment about the function type, so you can safely remove it from here.

Reply:
See, for example the `main` function from `/bin/ls`. Here how it looks with `asm.calls=false`:

![image](https://user-images.githubusercontent.com/203261/111106055-4dfa5100-858f-11eb-90a4-30dfa17bb676.png)

And now it has `asm.calls=true`:

![image](https://user-images.githubusercontent.com/203261/111106110-666a6b80-858f-11eb-8c77-2d2517490463.png)

So, could you please add a test for this case too, while you are on it?

Reply:
OK, I would do it later.